src/Atmos/MoistThermodynamics/states.jl

```diff
@@ -88,6 +88,25 @@ function LiquidIcePotTempSHumEquil(θ_liq_ice, q_tot, ρ, p)
     PhaseEquil(e_int, q_tot, ρ, T)
 end
 
+"""
+    LiquidIcePotTempSHumEquil_no_ρ(θ_liq_ice, q_tot, p)
+
+Constructs a [`PhaseEquil`](@ref) thermodynamic state from:
+
+ - `θ_liq_ice` - liquid-ice potential temperature
+ - `q_tot` - total specific humidity
+ - `p` - pressure
+"""
+function LiquidIcePotTempSHumEquil_no_ρ(θ_liq_ice, q_tot, p)
```

Comment:
can we just make it a 3-arg form of `LiquidIcePotTempSHumEquil`?

Reply:
Potentially, I need to check to see if the results/physics are the same for this case.

Reply:
Yeah, there's not a clear way to recover the density (corresponding to the saturation-adjusted temperature), given liquid potential temperature, q_tot and pressure.